src/ServiceControl.Audit.Persistence.RavenDb/UnitOfWork/RavenDbAuditUnitOfWorkFactory.cs

```diff
@@ -0,0 +1,23 @@
+﻿namespace ServiceControl.Audit.Persistence.RavenDb.UnitOfWork
+{
+    using Persistence.UnitOfWork;
+    using Raven.Abstractions.Data;
+    using Raven.Client;
+
+    class RavenDbAuditIngestionUnitOfWorkFactory : IAuditIngestionUnitOfWorkFactory
+    {
+        IDocumentStore store;
```

Comment:
by convention, we mostly move the fields to the bottom, so there's less noise as you open the file.